suplemon/logger.py

```diff
@@ -33,6 +33,8 @@ def close(self):
         self.acquire()
         try:
             for record in self.buffer:
+                if record.levelno < self.level:
```

Comment:
If we set a log level on a logger, then the record won't even be saved to `self.buffer` if it isn't at the proper threshold (or at least I think so).

Maybe we should set the log level to "info" and then only override with "debug" when specified in the config?